CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Added
+- Initialisation - Make liana initialisation use `opts.callback` if provided.
```

Comment:
[Dev Experience] Can we purpose a new major release that would expose a promise (vs the callback implementation)?
It would probably be much more "modern" than a callback.

If we release this new liana version as a v6, Lumber will still use liana v5 unless we implement the new version using a liana v6. It should work this way, right?

Reply:
Yes, we can do this.